assets/src/shared/lib/utils.ts

```diff
@@ -4,3 +4,25 @@ import { twMerge } from 'tailwind-merge';
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+const default_name_len =
+  window.innerWidth < 640
+    ? 35
+    : window.innerWidth < 1280
+      ? 55
+      : window.innerWidth < 1536
+        ? 50
+        : window.innerWidth < 1920
+          ? 15
+          : 40;
+
+export function useNameLen(input: string, postfix: string = ''): string {
```

Comment:
idk why this function do and what it's for
its looks like function for one very narrow case.

and please use function in this syntax  - `const fnName () => {}`

Reply:
useNameLen very non-intuitive title, use more clear name or use JSDoc

Reply:
this function looks is `костыль` when it didn't work out using CSS. Please try to use CSS for this, CSS is more optimization in this case as opposed to using JavaScript.

Reply:
> this function looks like костыль when it didn't work out using CSS. Please try to use CSS for this, CSS is more optimization in this case as opposed to using JavaScript.

Could you give me instructions on how I can do this with CSS, also it would be great to keep a feature that the erasure for the host and password would be of the form
host_with_long_name:888
to 
host_with_long...:888

Reply:
But... I'm not sure what good it'll do, maybe I can give it up.

Reply:
updated

rename 6a6978d7ce02bfa6068dff331ad85a8ef115d34c

convert to to arrow aa1346435d207808737d98908ade74fda22ba9dc